docs/en/interfaces/cli.md

```diff
@@ -158,6 +158,116 @@ $ clickhouse-client --param_tuple_in_tuple="(10, ('dt', 10))" -q "SELECT * FROM
 $ clickhouse-client --param_tbl="numbers" --param_db="system" --param_col="number" --query "SELECT {col:Identifier} FROM {db:Identifier}.{tbl:Identifier} LIMIT 10"
 ```
 
+## Connection string {#connection_string}
+
+The connection string for clickhouse-client is presented in URI format:
+
+```text
+clickhouse://[user_info@][hosts_and_ports][/dbname][?query_parameters]
+```
+
+where user_info is: `user[:password]`
+and hosts_and_ports is a list of values: `[host][:port],[host][:port]` Port is not mandatory.
+and query_parameters is a list of parameter[=value]: `param_name[=value]&param_name[=value]...` value may not be required for some of the parameters. Parameter names are case sensitive.
+
+Allowed query_parameters keys:
+
+- `secure` or shorthanded `s` - no value. If specified, client will connect to the server over a secure connection (TLS). See `secure` in [command-line-options](#command-line-options)
+
+These examples illustrate valid connection strings for clickhouse-client:
+
+```text
+clickhouse:
+clickhouse://localhost
+clickhouse://localhost:9000
+clickhouse://localhost/default
+clickhouse://default@localhost
+clickhouse://user:password@localhost
+clickhouse://user_name@localhost/some_database?secure
+clickhouse://host1:9000,host2:5000/some_database
+```
+
+The host component can either be an IP address or a host name. Put an IPv6 address in square brackets to specify it:
+
+```text
+clickhouse://[2001:db8::1234]
+```
+
+If user or/and password are not specified, default values will be used.
```

Comment:
A reader wonders what the default user / default password is?

Reply:
Documentation is changed